rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java

```diff
@@ -101,6 +106,18 @@ public boolean willClientHonorLease() {
       return SetupFrameFlyweight.honorLease(setupFrame);
     }
 
+    @Override
+    public boolean isResumeEnabled() {
+      return SetupFrameFlyweight.resumeEnabled(setupFrame);
+    }
+
+    @Override
+    public ByteBuf resumeToken() {
```

Comment:
Should this be null instead?  For things like metadata, null means not present.

Reply:
resume token cant be empty - there is no need to distinguish with null.  Looks like this change can be reduced to one token returning method - and empty would mean non-resumable

Reply:
@yschimke checking spec `Unsigned 16-bit integer of Resume Identification Token Length in bytes. (Not present if R flag is not set)`, having both `isResumeEnabled` and nullable `resumeToken` seems best option.